.devcontainer/devcontainer.json

```diff
@@ -20,11 +20,11 @@
 		"ms-dotnettools.csharp"
 	],
 
-	// Use 'postCreateCommand' to run commands after the container is created.
-	"postCreateCommand": "bash -i ${containerWorkspaceFolder}/.devcontainer/scripts/postCreateCommand.sh",
+	// Use 'onCreateCommand' to run pre-build commands inside the codespace
+	"onCreateCommand": "${containerWorkspaceFolder}/.devcontainer/scripts/onCreateCommand.sh",
 
 	// Add the locally installed dotnet to the path to ensure that it is activated
-	// This is needed so that things like the C# extension can resolve the correct SDK version
+	// This allows developers to just use 'dotnet build' on the command-line, and the local dotnet version will be used.
 	"remoteEnv": {
 		"PATH": "${containerWorkspaceFolder}/.dotnet:${containerEnv:PATH}",
```

Comment:
This line alone is super helpful. Is it possible for the user to create a new terminal window that doesn't inherit this -- in which case this should maybe go in `/.profile` or `~/.bash_profile`

But, it's fine for now.

Reply:
I've created a few new terminal windows, and they all get this in their $PATH. I don't believe this is needed to go into a `~/.bash_profile`.